images/linux/scripts/tests/Node.Tests.ps1

```diff
@@ -0,0 +1,14 @@
+Describe "Node.js" {
+    $binaries = @("node")
+    $module_commands = (Get-ToolsetContent).node_modules | ForEach-Object { $_.command }
+    $testCases = $binaries + $module_commands | ForEach-Object { @{NodeCommand = $_} }
+
+    It "<NodeCommand>" -TestCases $testCases {
+        param (
+            [string] $NodeCommand
+        )
+
+        "$NodeCommand --version" | Should -ReturnZeroExitCode
```

Comment:
As I understand, we have to add `command` to toolset because package name can be non-equal to command, right? 
if you have decided to put the command to toolset, let's also put the `--version` to toolset too to be more flexible.

Reply:
@maxim-lobanov I think putting `--version` to the toolset is out of the toolset purpose, maybe it is better to rename toolset fields to avoid confusion, like 
```
{
packageName: "netlify-cli",
cmdName: "netlify"
}
```

Reply:
You are correct, i do not think the semantic for "command" attribute should be "command to test" because, in the theory, the command can be used for other purposes. In fact you suggest to return to the "commandShouldRetunrZero" attribute i had in the very beginning :)
I think we should to live the the command so far and the third attribute like "testCommand" or something in case we would really need it

Reply:
@AlenaSviridenko i do not think we need to rename the fields as well
1. introducing another word "package" in addition to "node module" leads to the extra ambiguity for my opinion
2. we have pretty self explainable notation node_modules[].name and  node_modules[].command